spanner/google/cloud/spanner_admin_database_v1/gapic/database_admin_client.py

```diff
@@ -919,7 +817,7 @@ def get_iam_policy(
     def test_iam_permissions(
         self,
         resource,
-        permissions,
+        permissions=None,
```

Comment:
@busunkim96, @crwilcox  Can we get feedback to whomever is doing the proto annotations on this change?  The [`TestIamPermissionsRequest` docs](https://cloud.google.com/spanner/docs/reference/rpc/google.iam.v1#google.iam.v1.TestIamPermissionsRequest) say that permissions is a required field:

> permissions[] | string  REQUIRED: The set of permissions to check for  'resource'. Permissions with wildcards (such as '*', 'spanner.*',  'spanner.instances.*') are not allowed.

I've seen / commented on similar changes in other autogen PRs recently, as well.

Reply:
I don't understand why the codegen is making this optional when the [proto definition](https://github.com/googleapis/googleapis/blob/218164b3deba1075979c9dca5f71461379e42dd1/google/iam/v1/iam_policy.proto#L137) in googleapis clearly has a REQUIRED annotation on the field.

Reply:
Oh I know, we didn't update the yaml file. Doh! I'll send a change to @lukesneeringer.

Reply:
Same [issue in PHP](https://github.com/googleapis/google-cloud-php/pull/2438#issuecomment-553660325) and @dwsupplee says a fix has landed already. So we can close this PR and wait for the regen.

FYI @mbril.